frameworks/digital-outcomes-and-specialists/questions/briefs/evaluationTypeOutcomes.yml

```diff
@@ -2,20 +2,20 @@ name: Assessment methods
 id: evaluationType
 question: How you’ll assess suppliers
 question_advice: |
-  Choose how you want to assess suppliers.
-  
-  You can only use the methods you choose here.
-  
-  You must ask for a written proposal.
-  
+  All suppliers that reach the assessment stage will be asked to provide a written proposal.
```

Comment:
Yes, I think Roz must have edited my comment.  I didn't type the second paragraph.